src/lib/ppx_coda/ppx_representatives/runtime/ppx_representatives_runtime.ml

```diff
@@ -0,0 +1,27 @@
+module Util = struct
+  let rev_concat l =
+    let rec go acc l =
+      match l with
+      | [] ->
+          acc
+      | [] :: l ->
+          go acc l
+      | (x :: xs) :: l ->
+          go (x :: acc) (xs :: l)
+    in
+    go [] l
+end
+
+let unit_to_representatives = lazy [()]
+
+let bool_to_representatives = lazy [false]
+
+let int_to_representatives = lazy [0]
+
+let string_to_representatives = lazy [""]
+
+let char_to_representatives = lazy [' ']
+
+let list_to_representatives _ = lazy [[]]
+
```

Comment:
`bytes` is another built-in type

(so is `exn`, but I don't know what the representative would be)

`array` is another built-in type constructor

Reply:
int64 we may need -- I don't think we'll need any of the others at least for the rosetta work